adsingestp/parsers/base.py

```diff
@@ -153,6 +153,8 @@ def format(self, input_dict, format):
                      'edhist_acc': string
                      'edhist_rec': [string],
                      'edhist_rev': [string],
+                     'esources': [{'source': string,
+                                   'location': string}],
```

Comment:
You're actually storing it as a list of tuples, similar to `issn`, not a list of dicts